namerd/storage/k8s/src/main/scala/io/buoyant/namerd/storage/K8sV2DtabStoreInitializer.scala

```diff
@@ -0,0 +1,28 @@
+package io.buoyant.namerd.storage
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import io.buoyant.config.types.Port
+import io.buoyant.k8s.ClientConfig
+import io.buoyant.namerd.{DtabStore, DtabStoreConfig, DtabStoreInitializer}
+
+case class K8sConfigV2(
+  host: Option[String],
+  port: Option[Port],
+  namespace: Option[String]
+) extends DtabStoreConfig with ClientConfig {
+  @JsonIgnore
+  override def portNum: Option[Int] = port.map(_.port)
+
+  @JsonIgnore
+  override def mkDtabStore: DtabStore = {
+    val client = mkClient()
+    new CrdK8sDTabStore(client, dst, namespace.getOrElse(DefaultNamespace))
+  }
+}
+
+class K8sV2DtabStoreInitializer extends DtabStoreInitializer {
+  override def configClass = classOf[K8sConfigV2]
+  override def configId = "io.l5d.v2.k8s"
```

Comment:
Similarly to my above comment, I think this should be `io.l5d.k8s.crd`. We want the config id to make it clear that this is the CRD dtab store. Also, I think "k8s" should come _before_ the part of the config id that designates what _type_ of dtab store this is, since it's less specific.